libs/components/forms/src/lib/modules/checkbox/checkbox.component.ts

```diff
@@ -206,6 +227,27 @@ export class SkyCheckboxComponent implements ControlValueAccessor, OnInit {
     return this.#disabledChangeObs;
   }
 
+  /**
+   * Fires when the indeterminate state changes.
+   * @internal
+   */
+  @Output()
+  public get indeterminateChange(): Observable<boolean> {
```

Comment:
This Output was added to allow for two way binding. This is done so that we can turn off the `indeterminate` state whenever a checkbox is clicked both internally and externally.